CHANGELOG.md

```diff
@@ -5,6 +5,20 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### BREAKING CHANGES
+
+- renamed `RequestMiddlewareLogger` to `RequestMuxMiddlewareLogger` to distinguish between `fiber` and `mux`.
```

Comment:
can we avoid the breaking change? updating services to the v3 may become a useless pain 😞

Reply:
`RequestMiddlewareLogger` is ambiguous, now that we support multiple frameworks. Do you have better ideas than renaming?

Reply:
If it is also moved, the import will change so also a rename is not a problem. We could split middlewares in different packages, with every package a middleware. In this way, the function name could always be RequestMiddlewareLogger, but it will be imported by `some-path/middlewares/fiber` or `some-path/middlewares/gorilla`

I don't think it would be a pain, it is used only once in a service the function.

Reply:
the breaking makes sense since there are many changes to the usage interface.. that's ok i guess

Reply:
At the moment I moved everything middleware-related into a single package named `middleware`, so the two functions are named after the framework. Do you prefer the two packages solution or single one?